recipes/braggedgemodeling/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.1.1a3" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
The `name` variable appears to be undefined. Guessing this was an oversight. Could someone please submit a PR to correct this?

Reply:
Please see https://github.com/conda-forge/staged-recipes/pull/6533

Reply:
Thanks :)